app/scripts/store.js

```diff
@@ -220,6 +220,22 @@ var exports = (function() {
       });
     },
 
+    resetModel: function (modelContainer) {
+      return new Promise((resolve, reject) => {
+        if (modelContainer === undefined || modelContainer.id === undefined) {
```

Comment:
Why would we call this function without a modelContainer or with a modelContainer without an id?